Bottom: c0fb74247d6ef93d11a09422f4130eac2c89e77e
Top:    71652b21d274218c5e1bd87f67ab451633ecccc5
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 19:59:02 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
